charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -179,6 +179,13 @@ receivers:
       - type: container
         id: container-parser
         max_log_size: {{ $.Values.presets.logsCollection.maxRecombineLogSize }}
+      # combine multiline logs (e.g. stack traces)
+      - id: recombine-multiline
+        type: recombine
+        combine_field: body
+        is_first_entry: body matches "^[^\\s]"
+        source_identifier: attributes["log.file.path"]
+        max_log_size: {{ $.Values.presets.logsCollection.maxRecombineLogSize }}
```

Comment:
What happens to logs that don't need recombined? Does this add performance overhead?

Reply:
Regarding resources usage, we don't see any performance penalty. Below you can see the average CPU and memory usage before and after this add.

Before: 
<img width="2589" alt="image" src="https://github.com/user-attachments/assets/45956668-1150-4a66-9220-a6918ef243aa">

<img width="2589" alt="image" src="https://github.com/user-attachments/assets/26da9911-3c01-489a-8ff1-9a8beb433aef">

After:

<img width="2987" alt="image" src="https://github.com/user-attachments/assets/71d5fa43-6dfc-4f59-9da1-9688cd88fb52">

<img width="2593" alt="image" src="https://github.com/user-attachments/assets/ae48b666-a88f-4fd1-af5e-55b2dfd71652">

Is there any other metric that we can take an eye?

Does it make sense to add this using a feature toggle on values.yaml instead of force use it?

Reply:
Since this is a preset config, I'm trying to understand if `recombine` is universally helpful and wont impact any users who don't need it.  

If it negatively affects users who don't need to recombine logs, then it would be better for you to configure the filelog's operators manually instead of including it automatically in the preset's config.

Reply:
Technically isn't seems so, but I understand your point.

I still see value on this recombine or dynamically support custom operator additions to the Kubernetes filelog receiver without copying paste everything and lose future updates.

Let me also give you some possibilities to think about it:
- Add a feature toggle on values to add this block on this preset config
- Add a new hash on values with this new value as default to be used and anyone can make it empty if they want to disable
- Add a new hash on values to support custom operators for this Kubernetes filelog receiver with an empty value as default

Let me know what so you think about it